docs/reference/eql/search.asciidoc

```diff
@@ -42,5 +42,5 @@ GET sec_logs/_eql/search
 // TEST[continued]
```

Comment:
Github wont' let me comment on the snippet above, but it should be updated to something like:

```
[source,console]
----
GET sec_logs/_eql/search
{
  "event_type_field": "event.category",
 "rule": """
    process where process.name == "cmd.exe"
  """
----
}
```
I would leave `event.category` as the `event_type_field` as it conforms to ECS.

Reply:
Thanks @jrodewig - I'll follow-up with a different PR to fix the default `event_type_field` to `event.category` not just in docs, but in the tests as well.

Reply:
That works for me. Thanks!

Reply:
Looked at the code and figured that the `event_type_field` is actually implemented and your suggestions worked flawlessly.
Thanks!